vkcs/internal/acctest/acctest.go

```diff
@@ -195,3 +197,14 @@ func ImportStep(resourceName string) resource.TestStep {
 		ImportStateVerify: true,
 	}
 }
+
+func GenerateTags(testName string) map[string]string {
+	return map[string]string{
+		"TestName":    testName,
+		"CurrentTime": fmt.Sprint(time.Now().Unix()),
```

Comment:
Having a time in the resource name can be useful to make debugging easier, in this situation it would be better to make it human readable.

Reply:
done